.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+To help us get this pull request reviewed and merged quickly, please be sure to include the following items:
+
+* [ ] Tests (if applicable)
+* [ ] Documentation (if applicable)
+* [ ] Changelog entry
+* [ ] A full explanation here in the PR description of the work done
+
+## PR Type
+What kind of change does this PR introduce?
+
+* [ ] Bugfix
+* [ ] Feature
+* [ ] Code style update (formatting, local variables)
+* [ ] Refactoring (no functional changes, no api changes)
+* [ ] Build related changes
+* [ ] CI related changes
+* [ ] Documentation content changes
+* [ ] Tests
+* [ ] Other
+
+## Backward Compatibility
+
+Is this change backward compatible with the most recently released version? Does it introduce changes which might change the user experience in any way? Does it alter the API in any way?
+
+[ ] Yes (backward compatible)
+[ ] No (breaking changes)
```

Comment:
@mxhenry-godaddy My bad, I missed the list markers on these two. Last change, I swear! 😄 
```suggestion
* [ ] Yes (backward compatible)
* [ ] No (breaking changes)
```